cmd/micro/micro.go

```diff
@@ -472,9 +472,10 @@ func DoEvent() {
 	}
 
 	_, resize := event.(*tcell.EventResize)
-	if action.InfoBar.HasPrompt && !resize {
+	if action.InfoBar.HasPrompt || resize {
 		action.InfoBar.HandleEvent(event)
-	} else {
+	}
+	if !action.InfoBar.HasPrompt || resize {
```

Comment:
`make test` has revealed that this is buggy: when pressing Enter key in the command bar, `action.InfoBar.HandleEvent(event)` itself chnages the value of `action.InfoBar.HasPrompt` from true to false, so this Enter key event is wrongly sent to the buffer pane as well.